core/include/kernel/tpm.h

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
In header files, should be: `/* SPDX-License-Identifier: BSD-3-Clause */`

Reply:
Thanks for the comment. I was actually about to ask about this as I also had warnings before, when using '/* */' for the license. Didn't realize that it was different for .c and .h files.

Cheers